webapp/loggedin/modules/analysis/processingChain/processingChainState.js

```diff
@@ -0,0 +1,24 @@
+import * as R from 'ramda'
+
+import ObjectUtils from '../../../../../common/objectUtils'
+
+import * as AnalysisState from '../analysisState'
+
+export const stateKey = 'processingChainView'
```

Comment:
but why the state is ```{
analysis:{processingChainView:{processingChain}}
}``` ?

Can it just be ```analysis:{{processingChain}}``` ?

Reply:
right